src/qt/platformstyle.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "platformstyle.h"
+
+#include "guiconstants.h"
+
+#include <QApplication>
+#include <QColor>
+#include <QIcon>
+#include <QImage>
+#include <QPalette>
+#include <QPixmap>
+
+static const struct {
+    const char *platformId;
+    /** Show images on push buttons */
+    const bool imagesOnButtons;
+    /** Colorize single-color icons */
+    const bool colorizeIcons;
+    /** Extra padding/spacing in transactionview */
+    const bool useExtraSpacing;
+} platform_styles[] = {
+    {"macosx", false, false, true},
+    {"windows", true, false, false},
+    /* Other: linux, unix, ... */
+    {"other", true, true, false}
+};
+static const unsigned platform_styles_count = sizeof(platform_styles)/sizeof(*platform_styles);
```

Comment:
Why not? It's analogous to networkstyle.cpp, and it makes the code more readable by using a name instead of a long in-line expression.